src/main/java/de/retest/web/selenium/AutocheckingRecheckDriver.java

```diff
@@ -24,18 +24,20 @@
 	private final AutocheckingCheckNamingStrategy namingStrategy;
 
 	public AutocheckingRecheckDriver( final RemoteWebDriver wrapped ) {
+		this( wrapped, RecheckWebOptions.builder().build() );
+	}
+
+	@Deprecated
+	public AutocheckingRecheckDriver( final RemoteWebDriver wrapped, final RecheckOptions options ) {
 		super( wrapped );
+		this.options = options;
 		namingStrategy = new CounterCheckNamingStrategy();
```

Comment:
Automatically respect the default defined in `RecheckWebOptions`?

```suggestion
		namingStrategy = RecheckWebOptions.builder().build().getNamingStrategy();
```

Reply:
Ok, done.